pkg/manifest/manifest.go

```diff
@@ -0,0 +1,108 @@
+package manifest
+
+import (
+	"path/filepath"
+	"sync"
+	"time"
+
+	"github.com/ddev/ddev/pkg/globalconfig"
+	"github.com/ddev/ddev/pkg/manifest/storages"
+	"github.com/ddev/ddev/pkg/manifest/types"
+	"github.com/ddev/ddev/pkg/util"
+)
+
+func NewManifest(updateInterval time.Duration) *Manifest {
+	manifest := &Manifest{
+		fileStorage: storages.NewFileStorage(getLocalFileName()),
+		// TODO change to ddev repo before merge
+		githubStorage:  storages.NewGithubStorage(`gilbertsoft`, `ddev`, `manifest.json`, storages.Options{Ref: "task/upstream-infos"}),
```

Comment:
I think we should put the manifest.json somewhere else so that it's not tied to ddev/ddev? That would make it easier to update, and resolve the problem we're having here with having to use a temporary location. It would also make this testable on PRs, etc.

Reply:
Sure, we also can place it in a dedicated repo. The messages feature is the first one in the manifest and like written in the docs this could also be used for other stuff in the future. That's why I have chosen a generic name. But if we use a dedicated repo, we could change it to messages.json or sth similar.

At the moment the owner/repo/branch is hard coded but would make sense to make it configurable also for testing, true.

Reply:
Dedicated repo https://github.com/ddev/remote-config